cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,16 @@
+/// <reference types="cypress" />
+
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
```

Comment:
It would be better if you add some assertions to check if the user was generated correctly. This will ensure that the test data is correct before running the test.